pages/06_embedding_non_view_content.md

```diff
@@ -2,22 +2,64 @@
 title: Embedding Non-View Content
 ---
 
-Almost all embedded deployments of Tableau embed pre-built dashboards. But many embed other components of Tableau Server such as:
+Almost all embedded deployments of Tableau embed pre-built dashboards. But some embed other components of Tableau such as:
 
-* Web Authoring screens
+* web authoring views
 * The Tableau Server UI (embedded for integration and single sign-on reasons)
 
-There are no APIs to achieve this type of embedding, but you can use iframes and any of the [single sign-on capabilities]({{ site.baseurl }}/pages/02_auth_and_sso) to achieve this.
+The easiest and best way to include web authoring in your embedding scenario is to use the [Tableau Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html). The Tableau Embedding API v3 provides a web component, `<tableau-viz-authoring>`, which in just a few lines of HTML code, allows users to author the view in your web application. You can also use the Embedding API v3 to call JavaScript methods that let you dynamically configure and switch between the authoring or viewing experience.
```

Comment:
"allows users to author the view in your web application" => We should add if permissions allowed. 
"You can also use the Embedding API v3 to call JavaScript methods that let you dynamically configure and switch between the authoring or viewing experience" => Allow you to customize authoring lifecycle, for example you can dynamically configure and switch between the authoring or viewing experience

Reply:
done.